setup.cfg

```diff
@@ -75,9 +76,8 @@ extend-ignore =
     F811,  # support typing.overload decorator
     F722,  # allow Annotated[typ, some_func("some string")]
 exclude =
-    ui_*
     .tox
-    .venv
```

Comment:
I use .venv, you use venv - maybe we should just drop this entry all together as skeleton itself does not need it and you venv name is a personal choice.

Reply:
We do specify it in the docs as a local install options though

Reply:
OK, all good then.